libdeno/deno.h

```diff
@@ -48,6 +48,12 @@ void deno_delete(Deno* d);
 // Returns false on error.
 // Get error text with deno_last_exception().
 // 0 = fail, 1 = success
+//
+// TODO change return value to be const char*. On success the return
```

Comment:
Or maybe add an out param `const char** json_message`?
I'm not super fond if getting creative with return values, would rather just return 0 or -1.
The patterns I expect are that failure is indicated by a negative integer return value or a NULL pointer return value.